CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [9.1.2] - 2022.06.06
+### Fixed
```

Comment:
It is a missing feature not a broken one, isn't it?
For me it is "added" and minor version bump

Reply:
good point, fixed